### ISSUE TITLE
Onboarding: Remove Google MC and Google Ads Cards

### ISSUE THREAD

**Issue by joemcgill** [woocommerce/google-listings-and-ads#2565]
As part of #2509, we will be consolidating the UI for connecting to Google Merchant Center and Google Ads accounts into a unified UI with the `GoogleAccountCard`.

To begin this effort, we'll remove the existing `GoogleAdsAccountCard` and `GoogleMCAccountCard` components from the `SetupAccounts` flow of the onboarding flow. 

<img width="1079" alt="image" src="https://github.com/user-attachments/assets/19b92bf5-3d1f-4fcd-9d2e-b024dd0fa86e">

## Acceptance Criteria

- [ ] The Google Ads card is not shown on step one of onboarding
- [ ] The Google Merchant Center card is not shown on step one of onboarding
- [ ] The disclaimer text associated with the MC card `GoogleMCDisclaimer` is moved to the footer next to the FAQs (see screenshot).
- [ ] The continue button remains disabled even after the WP.com and Google cards are connected.
- [x] _UI changes showed in the Google card are out of scope of this issue._ 

## Implementation Brief

> [!NOTE]
> This work should be branched from [feature/2458-streamline-onboarding](https://github.com/woocommerce/google-listings-and-ads/tree/feature/2458-streamline-onboarding) and a PR should be created against the [feature/2458-consolidate-google-account-cards](https://github.com/woocommerce/google-listings-and-ads/tree/feature/2458-consolidate-google-account-cards) branch.

The `SetupAccounts` component that needs to be edited [is here](https://github.com/woocommerce/google-listings-and-ads/blob/5f2c5b10abfd5fe61a8956b5b3d3495825be280b/js/src/setup-mc/setup-stepper/setup-accounts/index.js#L149-L170).

## Test Coverage

- E2E tests in tests/e2e/specs/setup-mc/step-1-accounts.test.js are expected to fail until #2509 is complete.

**Comment by joemcgill** [comment]
@eason9487 would you mind validating this subtask? The goal of this one is just to set up the UI ahead of the consolidation effort being worked on via all of the #2509 tasks.

**Comment by joemcgill** [comment]
I don't think there is much risk in getting started with this, so I'm marking this as ready for work to unblock the start of this effort.

**Comment by eason9487** [comment]
Not sure how the original disconnect function for each account card will be consolidated into the new UI during onboarding?

**Comment by joemcgill** [comment]
@ankitguptaindia this is ready for QA. Note that this is a sub-task of #2509, so the onboarding flow will not be able to be completed until the whole feature is complete.

@eason9487
> Not sure how the original disconnect function for each account card will be consolidated into the new UI during onboarding?

All of the functionality handled by the separate cards will be handled within the consolidated UI being started in #2566 as part of #2509.

**Comment by eclarke1** [comment]
Engineering complete and approved, moving to UAT for @fblascogarma final approval

**Comment by eclarke1** [comment]
Just tagging @joemcgill and @dsawardekar  to advise this needs to be merged please

**Comment by mikkamp** [comment]
Closing this as completed since it was part of the [2.9 release](https://github.com/woocommerce/google-listings-and-ads/releases/tag/2.9.0).
